utils/vara-ui/src/components/checkbox/checkbox.module.scss

```diff
@@ -1,36 +1,198 @@
 .label {
   display: flex;
   align-items: center;
+  gap: 8px;
   cursor: pointer;
+  user-select: none;
```

Comment:
I understand that `user-select: none` probably prevents text from being selected on double click, but I think it can significantly affects UX. Sometimes there really can be a need to copy paste label's text.